libs/api/domains/financial-aid/src/lib/dto/currentApplication.ts

```diff
@@ -0,0 +1,7 @@
+import { Field, ObjectType } from '@nestjs/graphql'
+
+@ObjectType()
+export class CurrentApplicationResponse {
```

Comment:
This ObjectType should be namespaced. 

Eg `@ObjectType(‘LocalGovFinancialSupportApplicationStatus‘)`